coffea/processor/__init__.py

```diff
@@ -27,6 +27,13 @@
     defaultdict_accumulator,
     column_accumulator,
 )
+from ..nanoevents.schemas import (
```

Comment:
I know existing coffea code has relative imports but I'm trying to stick to absolute imports going forward